src/GameScannerManager.cs

```diff
@@ -338,7 +338,10 @@ public static bool RunFileQuickCheck(string gameFilePath, long expectedFileSize)
             if (File.Exists(tempFileName))
                 File.Delete(tempFileName);
 
-            var fileDownloader = new ChunkFileDownloader(fileInfo.HttpLink, tempFileName, GameScannerTempPath);
+            var fileDownloader = concurrentDownload == 1
+                ? (IFileDownloader) new SimpleFileDownloader(fileInfo.HttpLink, tempFileName)
+                : new ChunkFileDownloader(fileInfo.HttpLink, tempFileName, GameScannerTempPath,
+                    concurrentDownload);
```

Comment:
This is a pretty complicated statement now, split it up into several lines to make it more readable